### ISSUE TITLE
Status on hyepool?

### ISSUE THREAD

**Issue by ultra-pool** [bonesoul/hypepool#87]
@bonesoul Please give a status update of hypepool? is it working or dead? fully working with UI for a paid version or what?

Thank you

**Comment by pool2mine** [comment]
hi @bonesoul 

What's the status of this pool project? can i purchase the frontend? My priority is Ethereum. please let me know. Thanks!

**Comment by ultra-pool** [comment]
I guess all work has stopped on both pools

**Comment by bitcoinbrisbane** [comment]
I'm going to see if I can revive it over at https://github.com/bitcoinbrisbane/hypepool

**Comment by pool2mine** [comment]
@bitcoinbrisbane let me know if i can help, i finished up setting 2 separate pools, an open ethereum pool for ETH, and a yiimp for some cpu coins, now i’m building a 3rd one for monero, it would be cool to have them all together

**Comment by ultra-pool** [comment]
@bitcoinbrisbane does it have a working UI?

**Comment by bitcoinbrisbane** [comment]
Great guys / girls.  @ultra-pool I've put aside this week to see how good / bad it is, then can make the call on if Ill continue the project.

**Comment by ultra-pool** [comment]
Please keep us posted on this. Thank you

**Comment by pool2mine** [comment]
Hi @bitcoinbrisbane any news with this pool?

**Comment by bitcoinbrisbane** [comment]
This week I was looking at cfgminer and how it talks to the pool.  Will continue to work this week on it.  https://github.com/bitcoinbrisbane/hypepool

**Comment by pool2mine** [comment]
Thanks! and let me know if i can help with something, i still have a couple of free ip's and some space if you need a vm for testing

**Comment by bitcoinbrisbane** [comment]
I’ve removed all the shit coins, BTC only. Spun up BTC core, now just need a miner. Yesterday I hacked the bitmain firmware and turned on SSH. Can dedicate most of next week to this.

**Comment by pool2mine** [comment]
ok, for test only? i mean... are the shitcoins coming back? i needed ETH and XMR and maybe later some bitcoin clones

**Comment by bitcoinbrisbane** [comment]
Happy to do some major alts, like ETH, ETC, LTC etc.  But there was like 100 shit coins in there.

**Comment by pool2mine** [comment]
ok thanks! I have some AMD's and CPU's if you need some hash to test... just let me know!

**Comment by pool2mine** [comment]
hey @bitcoinbrisbane  hi, sorry to bother... any updates with this pool?

**Comment by bitcoinbrisbane** [comment]
Not really. I can’t get CGMiner to run on CPU to test. I’m looking to get a cheap USB miner so I can

**Comment by pool2mine** [comment]
I just set up monero for CPU mining on a cryptonote-nodejs-pool... the pool is a great piece of software, but... is there a way to add ethereum to supported coins? i really don't know how to do it. The lastest updated fork went private, I got a copy just before they hide it, if you think you can do it just let me know and i'll give you access.
